src/test/scala/db/UserDBUnitTest.scala

```diff
@@ -0,0 +1,73 @@
+package db
+
+import Dao.DAO
+import models.UserType.{Admin, Customer}
+import models.Users
+import org.mockito.Mockito.when
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatestplus.mockito.MockitoSugar.mock
+import java.util.UUID
+import scala.collection.mutable.ListBuffer
+
+class UserDBUnitTest extends AnyFlatSpec {
+
+  val userDB: DAO = mock[DAO]
+
+  it should "Add user should return new user added." in {
+    val user = Users(UUID.randomUUID(), "Jeet", 23, "gkp", "12/2/1998", Customer)
+    when(userDB.addUser(user)).thenReturn("new user added.")
```

Comment:
We do not use mocking if there are no dependent methods called.

Reply:
ok, so we should make object of that class, Right?
val userDB = new UserDB

Reply:
val userDB = new UserDB

  it should "Add user should return new user added." in {
    val user = Users(UUID.randomUUID(), "Jeet", 23, "gkp", "12/2/1998", Customer)
    assert(userDB.addUser(user) == "new user added.")
  }
  
  Will it be ok, for the above